src/components/Profile/components/SchedulePanel/index.jsx

```diff
@@ -76,28 +76,19 @@ const GordonSchedulePanel = (props) => {
 
   const loadData = async (searchedUser) => {
     try {
-      const scheduleControlInfo = await schedulecontrol.getScheduleControl(
-        searchedUser.AD_Username,
-      );
-      console.log({ scheduleControlInfo });
+      const profileInfo = await user.getProfileInfo(searchedUser.AD_Username);
       const schedule = await scheduleService.getSchedule(searchedUser.AD_Username, props.term);
+      setProfile(profileInfo);
       setEventInfo(scheduleService.makeScheduleCourses(schedule));
-      if (scheduleControlInfo) {
-        setDescription(
-          scheduleControlInfo.Description
-            ? // We decided to leave the regex code for now because the data stored in the database
-              // before changing the api is still in the regex form.
-              scheduleControlInfo.Description.replace(new RegExp('SlSh', 'g'), '/')
-                .replace(new RegExp('CoLn', 'g'), ':')
-                .replace(new RegExp('dOT', 'g'), '.')
-            : '',
-        );
-        setModifiedTimeStamp(scheduleControlInfo.ModifiedTimeStamp);
+      if (!profileInfo.PersonType?.includes('stu')) {
```

Comment:
Should be includes 'fac' (since  a faculty is also a student if auding a course).  Note that fac appears to actually be all regular employees (facstaff).